contracts/src/test/7683/T1XChainReaderTest.t.sol

```diff
@@ -8,15 +8,15 @@ import { OnchainCrossChainOrder } from "intents-framework/ERC7683/IERC7683.sol";
 
 import { T1XChainReader } from "../../libraries/xChain/T1XChainReader.sol";
 import { T1BasicSwapE2E } from "./T1BasicSwapE2E.t.sol";
-import { T1ERC7683Pull } from "../../7683/T1ERC7683Pull.sol";
+import { T1ERC7683 } from "../../7683/T1ERC7683.sol";
 
 contract T1XChainReaderTest is T1BasicSwapE2E {
     using TypeCasts for address;
 
     T1XChainReader internal originReader;
     T1XChainReader internal destinationReader;
-    T1ERC7683Pull internal L1T17683Pull;
-    T1ERC7683Pull internal L2T17683Pull;
+    T1ERC7683 internal L1T1ERC7683;
+    T1ERC7683 internal L2T1ERC7683;
```

Comment:
I would advocate for now used capital letters at the beginning of variable names